source/server/admin/admin.cc

```diff
@@ -449,8 +456,16 @@ Http::Code AdminImpl::handlerClusters(absl::string_view url,
 }
 
 void AdminImpl::addAllConfigToDump(envoy::admin::v3::ConfigDump& dump,
-                                   const absl::optional<std::string>& mask) const {
-  for (const auto& key_callback_pair : config_tracker_.getCallbacksMap()) {
+                                   const absl::optional<std::string>& mask,
+                                   const bool include_eds) const {
+  Envoy::Server::ConfigTracker::CbsMap callbacks_map = std::move(config_tracker_.getCallbacksMap());
```

Comment:
Are you sure you want to `std::move` here?

Reply:
What happens if you call this multiple times?

Reply:
Removed the `std::move`